packages/client-sdk-web/src/index.ts

```diff
@@ -119,6 +120,11 @@ import {
   TopicClientConfiguration,
 } from './config/topic-configuration';
 
+// VectorClient Response Types
+import * as CreateIndex from '@gomomento/sdk-core/dist/src/messages/responses/create-index';
+import * as ListIndexes from '@gomomento/sdk-core/dist/src/messages/responses/list-indexes';
+import * as DeleteIndex from '@gomomento/sdk-core/dist/src/messages/responses/delete-index';
```

Comment:
Michael I believe that you decided we wanted to start namespacing these a bit more in python. Do you have thoughts on what we should do here?